protoc-gen-insomniaenv/main.go

```diff
@@ -129,12 +131,13 @@ func (e *insomniaenv) generate(file *descriptor.FileDescriptorProto) *plugin.Cod
 
 	insomniaExport.Resources = resources
 
-	b, err := json.MarshalIndent(insomniaExport, "", "	")
+	b, err := json.MarshalIndent(insomniaExport, "", "\t")
 	if err != nil {
 		return nil
 	}
 
-	resp.Name = proto.String(file.GetName() + "-insomnia-env.json")
+	fileWithoutPath := strings.TrimSuffix(file.GetName(), filepath.Ext(file.GetName()))
+	resp.Name = proto.String(fmt.Sprintf("%s-insomnia-env.json", fileWithoutPath))
```

Comment:
`bla.proto-insomnia-env.json` is dumb, removing the double dot. Now this is `bla-insomnia-env.json`